pyproject.toml

```diff
@@ -21,12 +21,17 @@ fastapi = "0.97.0"
 uvicorn = "0.22.0"
 influxdb-client = "1.37.0"
 langid = "1.1.6"
-torch = {version = "^2.0.1+cpu", source = "pytorch"}
+torch = [
+    { version = "^2.0.1", source = "pypi", platform = "darwin" },
+    { version = "^2.0.1+cpu", source = "pytorch", platform = "linux" },
+    { version = "^2.0.1+cpu", source = "pytorch", platform = "win32" },
+]
 bertopic = "^0.15.0"
 llvmlite = "^0.40.1"
 python-dotenv = "^1.0.0"
 python-dateutil = "^2.8.2"
 transformers = "4.36.0"
+pillow = "10.2.0"
```

Comment:
Why do we need pillow?

Reply:
Upd: I merged PR from dependabot. This change is not needed.
I suggest to remove this line because we do not use pillow directly.

Reply:
I had to fix https://avd.aquasec.com/nvd/2023/cve-2023-50447/ by upgrading pillow version